src/panel/components/Background.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 export const Background = styled.div`
   overflow: hidden;
   position: fixed;
-  background-color: ${(props) => props.theme.dark[0]};
+  background-color: ${(p) => p.theme.canvas.base};
```

Comment:
Should we keep this named `props` just for readability sake?

Reply:
Yeah it's a good point! I noticed the majority of styled components used `p` instead of `props` so changed it for consistencies sake but I would also vouch for `props` or even destructuring the object `${({ theme }) => theme.canvas.base};`

Reply:
Just weighing in on this, I think all of the above seem totally fine to me!

I'm surprised at how much time I've saved by just writing `p => p.theme` instead of destructuring, it all adds up I guess 😅